### PR TITLE
[MH2018-31] Feature/sh product list

### PR DESCRIPTION
**Pull request by jaimelr**
Created a page to show the products of a collection:
Flat:
collections

The development store:
https://fiorella-dev-2.myshopify.com

### CODE REVIEW

File: fiorella/src/styles/pages/collection.scss [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  padding: 0 10px;
+  text-align: center;
+
+  .return-button-container {
+    font-family: $font-default;
+    margin: 5px 0 0;
+    text-align: right;
+    width: 100%;
+
+    a {
```
Comment (magmahound) [inline fiorella/src/styles/pages/collection.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/pages/collection.scss [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    margin-top: 5px;
+  }
+}
+
+.collection-header-banner {
+  display: block;
+  width: 100%;
+
+  img {
+    width: 100%;
```
Comment (magmahound) [inline fiorella/src/styles/pages/collection.scss]: Properties should be ordered height, width

File: fiorella/src/sections/collection-hero.liquid [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<div class="collection-header-banner">
+  {% for block in section.blocks %}
+    {{ block.settings.collection-hero | img_url: '800x800' | img_tag }}
+  {% endfor %}
+</div>
+
+{% schema %}
+{
+  "name":       "Collection hero",
```
Comment (IsmaelVillavicencio) [inline fiorella/src/sections/collection-hero.liquid]: there are spaces

File: fiorella/src/sections/collection-hero.liquid [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "max_blocks": 2,
+  "blocks": [
+    {
+      "type": "image",
+      "name": "Collection hero",
+      "settings": [
+        {
+          "type": "image_picker",
+          "id":   "collection-hero",
```
Comment (IsmaelVillavicencio) [inline fiorella/src/sections/collection-hero.liquid]: there is an space

File: fiorella/src/sections/collection-hero.liquid (hunk 3 above)
Comment (Ilyeo) [reply fiorella/src/sections/collection-hero.liquid]: can you indent correctly, please?

File: fiorella/src/sections/collection-hero.liquid (hunk 4 above)
Comment (Ilyeo) [reply fiorella/src/sections/collection-hero.liquid]: > can you indent correctly, please?

same here...

File: fiorella/src/snippets/products.liquid [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    </div>
+
+  {% else %}
+    {% if collection.handle == 'all' and collection.all_vendors.size == 0 and collection.all_types.size == 0 %}
+      {% for i in (1..8) %}
+        <a href="#">
+          {% capture current %}{% cycle 1, 2, 3, 4, 5, 6 %}{% endcapture %}
+          {{ 'product-' | append: current | placeholder_svg_tag: 'placeholder-svg placeholder-svg--small' }}
+        </a>
+
```
Comment (rubiocanino) [inline fiorella/src/snippets/products.liquid]: Unncesary space

File: fiorella/src/snippets/products.liquid [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        <p>
+          <a href="#">{{ 'homepage.onboarding.product_title' | t }}</a>
+        </p>
+
```
Comment (rubiocanino) [inline fiorella/src/snippets/products.liquid]: Unncesary space

**Pull request merged**